package.json

```
@@ -12,7 +12,7 @@
     "@babel/node": "^7.13.0",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/preset-env": "^7.13.5",
-    "@hathor/wallet-lib": "^1.0.3",
+    "@hathor/wallet-lib": "^1.0.4",
```

Since we're using "^", npm will already install 1.0.4 (if it's the latest)

Actually, it will install any version that is <2.0.0

Should we fix the version here? 

```suggestion
    "@hathor/wallet-lib": "1.0.4",
```

---

I think we should lock the wallet-lib version, but i'll make this on another PR.
There is a ktlo to start using some best practices, i'll add this on that PR.